net-im/slack++/slack++-0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Enterprise chat and services on the command line - slack++"
+DESCRIPTION="slack++ is an ncurses based client the Searchable Log of All Conversation and Knowledge"
+HOMEPAGE="https://github.com/ga2arch/slack"
+COPYRIGHT="2018"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="961dec741764dfb81999703b84989d93ec7a88ee"
+SOURCE_URI="https://github.com/ga2arch/slack/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9c84b5d0ee7b528edfdde219263dd7c307c6e8cd9aadc491e7148f7457ea30a5"
+SOURCE_DIR="slack-$srcGitRev"
+#PATCHES=""
+
+ARCHITECTURES="x86 !x86_gcc2"
```

Comment:
If it builds with gcc7 you probably could add x86_64 in there too so someone could check it there for you